tests/unit/json_formatter_test.cc

```diff
@@ -36,7 +36,13 @@ SEASTAR_TEST_CASE(test_simple_values) {
     BOOST_CHECK_EQUAL("3.5", formatter::to_json(3.5));
     BOOST_CHECK_EQUAL("true", formatter::to_json(true));
     BOOST_CHECK_EQUAL("false", formatter::to_json(false));
-    BOOST_CHECK_EQUAL("\"apa\"", formatter::to_json("apa"));
+    BOOST_CHECK_EQUAL("\"apa\"", formatter::to_json("apa")); // to_json(const char*)
+
+    using namespace std::string_literals;
+    auto str = "\0 COWA\bU\nGA [{\r}]\x1a"s,
+         expected = "\"\\u0000 COWA\\bU\\nGA [{\\r}]\\u001A\""s;
+    BOOST_CHECK_EQUAL(expected, formatter::to_json(str)); // to_json(const sstring&)
+    BOOST_CHECK_EQUAL(expected, formatter::to_json(str.c_str(), str.size())); // to_json(const char*, size_t)
```

Comment:
Please also add a test for a string that does **not** have any special characters, because this invokes a very different code path.

Reply:
Done.